promptui.go

```diff
@@ -1,18 +1,89 @@
-// Package promptui provides ui elements for the command line prompt.
+// Package Promptui is a library providing a simple interface to create command-line prompts for go.
```

Comment:
*promptui, lowercase.

The convention is more like the original sentance, but you could word it like:
`Package promptui provides a simple interface for creating command line prompts.`